### PR TITLE
[Numpy] random.randint() implemented

### PR DESCRIPTION
**Pull request by xidulu**
## Description ##
np.random.randint()
https://docs.scipy.org/doc/numpy-1.15.1/reference/generated/numpy.random.randint.html

### Essentials ###
Please feel free to remove inapplicable items for your PR.
- [ ] The PR title starts with [MXNET-$JIRA_ID], where $JIRA_ID refers to the relevant [JIRA issue](https://issues.apache.org/jira/projects/MXNET/issues) created (except PRs with tiny changes)
- [ ] Changes are complete (i.e. I finished coding on this PR)
- [ ] All changes have test coverage:
- Unit tests are added for small changes to verify correctness (e.g. adding a new operator)
- Nightly tests are added for complicated/long-running ones (e.g. changing distributed kvstore)
- Build tests will be added for build configuration changes (e.g. adding a new build option with NCCL)
- [ ] Code is well-documented: 
- For user-facing API changes, API doc string has been updated. 
- For new C++ functions in header files, their functionalities and arguments are documented. 
- For new examples, README.md is added to explain the what the example does, the source of the dataset, expected performance on test set and reference to the original paper if applicable
- Check the API doc at http://mxnet-ci-doc.s3-accelerate.dualstack.amazonaws.com/PR-$PR_ID/$BUILD_ID/index.html
- [ ] To the my best knowledge, examples are either not affected by this change, or have been fixed to be compatible with this change

### Changes ###
- [ ] Feature1, tests, (and when applicable, API doc)
- [ ] Feature2, tests, (and when applicable, API doc)

## Comments ##
- If this change is a backward incompatible change, why must this change be made.
- Interesting edge cases to note here

### CODE REVIEW

File: python/mxnet/ndarray/numpy/random.py [hunk 1]
```diff
@@ -20,7 +20,67 @@
...
+        from the distribution (see above for behavior if ``high=None``).
+    size : int or tuple of ints, optional
+        Output shape.  If the given shape is, e.g., ``(m, n, k)``, then
+        ``m * n * k`` samples are drawn.  Default is None, in which case a
+        single value is returned.
+    dtype : dtype, optional
+        Desired dtype of the result. All dtypes are determined by their
+        name, i.e., 'int64', 'int', etc, so byteorder is not available
+        and a specific precision may have different C types depending
+        on the platform. The default value is 'np.int'.
```
Comment (haojin2) [inline python/mxnet/ndarray/numpy/random.py]: Remember to add `ctx` and `out` to the `Parameters` section, same for all other docs for this op

File: ci/docker/runtime_functions.sh [hunk 2]
```diff
@@ -542,7 +542,6 @@ build_ubuntu_cpu_clang60_mkldnn() {
     build_ccache_wrappers
 
     make \
-        ENABLE_TESTCOVERAGE=1         \
```
Comment (marcoabreu) [inline ci/docker/runtime_functions.sh]: Please don't do that.

File: ci/jenkins/Jenkins_steps.groovy [hunk 3]
```diff
@@ -389,7 +389,7 @@ def compile_unix_clang_6_mkldnn_cpu() {
           timeout(time: max_time, unit: 'MINUTES') {
             utils.init_git()
             utils.docker_run('ubuntu_cpu', 'build_ubuntu_cpu_clang60_mkldnn', false)
-            utils.pack_lib('mkldnn_cpu_clang6', mx_mkldnn_lib, true)
+            utils.pack_lib('mkldnn_cpu_clang6', mx_mkldnn_lib, false)
```
Comment (marcoabreu) [inline ci/jenkins/Jenkins_steps.groovy]: Nope.

File: ci/docker/runtime_functions.sh (hunk 2 above)
Comment (reminisce) [reply ci/docker/runtime_functions.sh]: @marcoabreu `TESTCOVERAGE` leads to very large binary size resulting in linking error. What's the practical way can you recommend for now? We cannot just stop adding more operators.

Comment (marcoabreu) [reply ci/docker/runtime_functions.sh]: Agree. We'll investigate the linking error today.

**Comment by haojin2** [comment]
@marcoabreu the changes to the ci files are reverted now, please kindly review again and raise your request for change if it looks good to you.

**Pull request merged**